packages/passport/sdk-sample-app/src/components/zkevm/EthSendTransactionExamples/CallSmartContract.tsx

```diff
@@ -0,0 +1,161 @@
+import React, {
+  useCallback, useEffect, useState,
+} from 'react';
+import { Accordion, Form } from 'react-bootstrap';
+import InputGroup from 'react-bootstrap/InputGroup';
+import { usePassportProvider } from '@/context/PassportProvider';
+import WorkflowButton from '@/components/WorkflowButton';
+import { RequestExampleProps } from '@/types';
+import { useImmutableProvider } from '@/context/ImmutableProvider';
+import { ethers } from 'ethers';
+import * as encUtils from 'enc-utils';
+
+function CallSmartContract({ disabled, handleExampleSubmitted }: RequestExampleProps) {
+  const { orderbookClient } = useImmutableProvider();
+  const defaultAddress = orderbookClient.config().seaportContractAddress;
+  const [fromAddress, setFromAddress] = useState<string>('');
+  const [toAddress, setToAddress] = useState<string>(defaultAddress);
+  const [toAddressError, setToAddressError] = useState<string>('');
+  const [data, setData] = useState<string>('123');
```

Comment:
```suggestion
  const [data, setData] = useState<string>('1234563343333');
```
the data have to be longer than 10 length. Otherwise, it will return the method_id, which is wrong from the passport MR. 
so for the default data, please put it larger than 10 in length.

You also need to check if the user provides more than 10-length data. 


But to sample the whole process, you can just hardcode the data and alway use the hardcoded data instead of allow user to do the input.